services/homepage.go

```diff
@@ -0,0 +1,37 @@
+
+
+package services
+
+import (
+"net/http"
+"io/ioutil"
+"path/filepath"
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/middleware"
+)
+
+const contentTypeHeader = `Content-Type`
+const htmlType = `text/html`
+
+func StartUiMode() {
+	r := chi.NewRouter()
+	r.Use(middleware.Logger)
+	r.Use(middleware.Recoverer)
+
+	r.Mount("/", getIndexPage())
+
+	http.ListenAndServe(":9191", r)
+}
+
+func getIndexPage() *chi.Mux {
+	router := chi.NewRouter()
+
+	router.Get("/", func(w http.ResponseWriter, r *http.Request) {
+		absPath, _ := filepath.Abs("./ui/index.html")
```

Comment:
always catch errors (i didn't in my example), but in production code, you should always handle them. if there is an error, log it and do not proceed with the next step. e.g. 

absPath, err := filepath.Abs("./ui/index.html")


if err != nil {
   logrus.Error("Unable to find homepage", err)
   w.WriteHeader(http.InternalServerError)
    return
}